system/Model.php

```diff
@@ -102,6 +102,13 @@ class Model
 	 */
 	protected $insertID = 0;
 
+	/**
+	 * Indicates if the IDs are auto-incrementing.
+	 *
+	 * @var boolean
+	 */
+	public $incrementing = true;
```

Comment:
At the moment the models assume an auto-incrementing id for every table. Please stop trying to revise that. If it's something you truly feel needs addressed again, do so in a separate PR so I don't have to totally kill this one, since they're unrelated. Remove this feature/check.

Reply:
you're complety right, i will address this in a seperate pr, im sorry.